src/chttpd/src/chttpd_rewrite.erl

```diff
@@ -71,8 +71,9 @@ do_rewrite(#httpd{mochi_req=MochiReq}=Req, {Props}=Rewrite) when is_list(Props)
                 undefined -> erlang:get(mochiweb_request_body);
                 B -> B
             end,
+            NewMochiReq:cleanup(),
             case Body of
-                undefined -> NewMochiReq:cleanup();
+                undefined -> [];
                 _ -> erlang:put(mochiweb_request_body, Body)
```

Comment:
Are you sure this has to be backported? It's not elixir

Reply:
Yes, it is a minor fix detected during the rewrite_js porting. rewrite_js_test.exs will fail without this fix.

It is already merged into master

Reply:
Ok then!